userspace/libsinsp/sinsp.cpp

```diff
@@ -546,6 +544,9 @@ void sinsp::open_common(scap_open_args* oargs)
 		oargs->proc_callback_context = this;
 	}
 	oargs->import_users = m_usergroup_manager.m_import_users;
+	// We need to subscribe to container manager notifiers beore
+	// scap starts scanning proc.
+	m_usergroup_manager.init();
```

Comment:
As privately discussed, we may want to rename `init` to something that makes it clear it's subscribing to container manager.

Reply:
Fixed in latest commit! Thanks!